aravis.spec

```diff
@@ -20,16 +20,21 @@ BuildRequires:	pkgconfig(gtk+-3.0)
 BuildRequires:	pkgconfig(libnotify)
 BuildRequires:	pkgconfig(gstreamer-base-1.0) >= 1.0
 BuildRequires:	pkgconfig(gstreamer-app-1.0)
+BuildRequires:	pkgconfig(libusb-1.0)
+BuildRequires:	pkgconfig(libcap-ng)
+BuildRequires:	pkgconfig(audit)
 
 Source10:	aravis.png
 
-%global fullname %{name}-0.4
+# I don't know why version 0.5.7 is actually 0.6
```

Comment:
0.6 is the API version, and 0.5.7 is the version number of the development series. The stable series will have 0.6.x versions.